.github/workflows/conformance-eks.yaml

```
@@ -107,6 +108,11 @@ jobs:
           filters: |
             src:
               - '!(test|Documentation)/**'
+            eni:
+              - 'pkg/aws/**'
+              - 'pkg/ipam/**'
+              - 'pkg/policy/groups/aws/**'
+              - 'vendor/github.com/aws/aws-sdk-go-v2/**'
```

Same as for `conformance-eks-v1.11.yaml`:

```suggestion
              - 'vendor/github.com/aws/**'
```